web/react/stores/user_store.jsx

```diff
@@ -164,13 +164,25 @@ var UserStore = assign({}, EventEmitter.prototype, {
     BrowserStore.setItem("sessions", sessions);
   },
   getSessions: function() {
-    return BrowserStore.getItem("sessions", []);
+    var sessions = BrowserStore.getItem("sessions");
+
+    if (sessions == null) {
+        return BrowserStore.getItem("sessions", {loading: true});
```

Comment:
Ohhhh, ok so if it's undefined, _then_ make it that value, my mistake totally misunderstood that, thanks for letting me know.